script/utils/ExistingDeploymentParser.sol

```diff
@@ -116,6 +117,7 @@ contract ExistingDeploymentParser is Script, Test {
         operationsMultisig = stdJson.readAddress(existingDeploymentData, ".parameters.operationsMultisig");
         communityMultisig = stdJson.readAddress(existingDeploymentData, ".parameters.communityMultisig");
         pauserMultisig = stdJson.readAddress(existingDeploymentData, ".parameters.pauserMultisig");
+        timelock = stdJson.readAddress(existingDeploymentData, ".parameters.timelock");
```

Comment:
I see this is added here, but don't see a matching line added to `_parseInitialDeploymentParams`. Is that important?